server/src/internalClusterTest/java/org/elasticsearch/search/sort/FieldSortIT.java

```diff
@@ -332,34 +340,22 @@ public void test3078() {
         refresh();
         assertResponse(
             prepareSearch("test").setQuery(matchAllQuery()).addSort(SortBuilders.fieldSort("field").order(SortOrder.ASC)),
-            response -> {
-                assertThat(response.getHits().getAt(0).getSortValues()[0].toString(), equalTo("1"));
-                assertThat(response.getHits().getAt(1).getSortValues()[0].toString(), equalTo("10"));
-                assertThat(response.getHits().getAt(2).getSortValues()[0].toString(), equalTo("100"));
-            }
+            assertConsumer
```

Comment:
I have not grouped the `assertRespone` methods in this test because of the index updates in between. I found it less boilerplate to group and reuse the Consumer object